cmd/immuadmin/command/database.go

```diff
@@ -33,10 +33,12 @@ import (
 func addDbUpdateFlags(c *cobra.Command) {
 	c.Flags().Bool("exclude-commit-time", false,
 		"do not include server-side timestamps in commit checksums, useful when reproducibility is a desired feature")
-	c.Flags().Bool("replication-enabled", false, "set database as a replica")
+	c.Flags().Bool("replication-enabled", false, "set database as a replica") // TODO: flag name should be changed to something like `replication-is-replica`
+	c.Flags().Bool("replication-sync-replication", false, "enable synchronous replication")
+	c.Flags().Uint32("replication-sync-followers", 0, "set a minimum number of followers for ack replication before transactions can be committed")
```

Comment:
I'd think from perspective of the deployment, the name `replication-sync-followers` is clear because it means how many followers are actually synchronized with the master. There could be more followers, but that specify the number which is guaranteed to be in sync
However I got your point, we may need to get feedback on this as there are multiple possibilities @SimoneLazzaris ?